sdk/textanalytics/ai-text-analytics/test/utils/recordedClient.ts

```diff
@@ -46,25 +35,33 @@ export const environmentSetup: RecorderEnvironmentSetup = {
   queryParametersToSkip: []
 };
 
-export function createRecordedClient(
-  context: Context,
-  apiKey?: AzureKeyCredential
-): RecordedClient {
-  const recorder = record(context, environmentSetup);
+export type AuthMethod = "APIKey" | "AAD";
 
+export function createClient(
```

Comment:
I would add a quick doc comment explaining that `createRecorder` or `record`  needs to be called first. That was originally the point of `createRecordedclient`: to enforce the ordering of creating the client after the recorder.